src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -278,9 +278,12 @@ private void writeErrorResponse(int status, String message)
 
         final DirectBuffer payload = new UnsafeBuffer(payloadText.toString().getBytes(StandardCharsets.UTF_8));
 
-        int writableBytes = Math.min(correlation.state().budget, payload.capacity());
+        ServerAcceptState acceptState = correlation.state();
+        int writableBytes = Math.min(
+                acceptState.acceptReplyWindowBudget - acceptState.acceptReplyWindowPadding, payload.capacity());
         if (writableBytes > 0)
         {
+            acceptState.acceptReplyWindowBudget -= writableBytes + acceptState.acceptReplyWindowPadding;
             factory.writer.doData(target, targetId, payload, 0, writableBytes);
         }
         if (writableBytes < payload.capacity())
```

Comment:
Can `writableBytes` be negative now, for example if `acceptReplyWindowPadding > acceptReplyWindowBudget`?